src/Microsoft.Diagnostics.Monitoring.StartupHook/ParameterCapturing/Eventing/ParameterCapturingEventSource.cs

```diff
@@ -123,10 +123,12 @@ public void CapturedParameterStart(
             int managedThreadId,
             string methodName,
             string methodModuleName,
-            string methodDeclaringTypeName
+            string methodDeclaringTypeName,
+            uint methodToken,
```

Comment:
These are fixed-length values, so they should be placed before the variable-length ones to make event source more efficient (e.g. place them before the first `string`). While this would technically a breaking change in the event source contract (appending isn't), we're the only consumer of this so we can safely do it.